### DISCUSSION TITLE
Resuming Quiz Progress in the QUML Player

### DISCUSSION THREAD

**Discussion by rushi-tekdi** [Sunbird-Knowlg/Community#229] [Q&A]
In the QUML player, if a user closes a test midway, we would like all previous answers to be saved and for the test to resume from where they left off, including the time remaining.

Is this feature available in the QUML player? If so, please provide the steps and configurations required to enable this functionality.

GitHub Repository:
[Sunbird QUML Player](https://github.com/project-sunbird/sunbird-quml-player)

@coolbung @pallakartheekreddy @snehal0904

**Comment by pallakartheekreddy** [comment]
@rushi-tekdi As of now, the QuML player does not natively support a resume feature that allows users to continue a quiz from where they left off.
